src/compilers.coffee

```diff
@@ -326,6 +326,12 @@
   target: 'Machine Code'
   type: 'Native'
   url: 'http://landinghub.visualstudio.com/visual-cpp-build-tools'
+,
+  name: 'Scala Natvie'
+  source: 'Scala'
+  target: 'Machine Code'
```

Comment:
I'd say it targets LLVM IR which transitively targets Machine Code.

Reply:
changed to Intermediate and LLVM IR :) Thx for comment